README.md

```diff
@@ -91,11 +84,13 @@ See also [Docker](https://cardano-foundation.github.io/cardano-wallet/user-guide
 
 NixOS users can also use the [NixOS service](https://cardano-foundation.github.io/cardano-wallet/user-guide/NixOS).
 
-## How to install (Linux / Windows / Mac OS)
+## Obtaining `cardano-wallet`
+
+### Executables (Linux / Windows / Mac OS)
 
-See **Installation Instructions** for each available [release](https://github.com/input-output-hk/cardano-wallet/releases).
+We provide executables as part of our [releases](https://github.com/input-output-hk/cardano-wallet/releases), see also the installation instructions.
```

Comment:
Since the phrase following the comma is a verb phrase, I'd suggest making it a separate sentence. (Alternatively, you could use a semi-colon.)

Additionally, shouldn't we replace "input-output-hk" with "cardano-foundation" in the URL?

```suggestion
We provide executables as part of our [releases](https://github.com/cardano-foundation/cardano-wallet/releases). See the installation instructions for more details.
```

Reply:
I will change the link.
I will use a semicolon — the "for more details" just lengthens the sentence without adding information.

Reply:
> I will use a semicolon — the "for more details" just lengthens the sentence without adding information.

> "We provide executables as part of our releases; see also the installation instructions."

This still sounds a little unnatural to me.

The point of adding "for more details" was to make it a more natural sentence (and not to add useless information 😄).

But if you feel that's too long, then perhaps we can write:

> We provide executables as part of our [releases](https://github.com/cardano-foundation/cardano-wallet/releases). Please also see the [installation instructions](link).

Reply:
I have changed it to

> We provide executables as part of our [releases](https://github.com/cardano-foundation/cardano-wallet/releases). Please also see the installation instructions highlighted in the release notes.

— there is no unique link for the installation instructions.